ONSdigital/census-rh-ui
# Cr 1326 transient uac journey

# Motivation and Context
New Transient UAC journey

# What has changed
Introduces new 'transient' UAC journey.
If estabType of entered UAC is set as 'Transient' the user is asked to enter their nearest Town, and then to select their accommodation type. These are then combined and fed to EQ in the 'display_address' value.
Code has normal error trapping and required unit tests

No Cucumber updates are required to merge, although specific tests may need building

# How to test?
'Create' a transient UAC, and enter it's UAC into /en/start/
Confirm get new journey

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1303

# Screenshots (if appropriate):
